### PR TITLE
feat(angular): add setup-ssr generator

### PR DESCRIPTION
**Pull request by Coly010**
## Current Behavior

We do not currently have a generator to add an SSR setup to an Angular app

## Expected Behavior

We should have a generator to add this behaviour

### CODE REVIEW

File: packages/angular/src/generators/setup-ssr/schema.json [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        "$source": "argv",
+        "index": 0
+      },
+      "x-prompt": "What app would you like to generate an Angular Universal configuration for?",
+      "x-dropdown": "projects"
+    },
+    "appId": {
+      "type": "string",
+      "format": "html-selector",
+      "description": "The appId to use withServerTransition.",
```
Comment (leosvelperez) [inline packages/angular/src/generators/setup-ssr/schema.json:22]:
```suggestion
      "description": "The `appId` to use with `withServerTransition`.",
```

**Review by leosvelperez — APPROVED** [review]
LGTM! 🎉 

Just left a minor suggestion.

**Comment by sir-captainmorgan21** [comment]
@Coly010 is this available in an existing version? Im on 15.0.4 and dont see it

**Comment by Coly010** [comment]
Yes, I can confirm it's in an existing version 
image

**Pull request merged**